stable/redis/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 9.2.0
+version: 9.2.1
```

@rjasper-frohraum, with this commit https://github.com/helm/charts/commit/e413679993ae5618cf94bf1386cbd7f82537b3d0, the chart version got increased so you will need to bump it again. You may want to rebase your branch.

```suggestion
version: 9.2.2
```

---

Done.